railties/test/generators/named_base_test.rb

```diff
@@ -6,6 +6,8 @@
 module ActiveRecord
   class Base
     class << self
+      undef pluralize_table_names
+      undef pluralize_table_names=
       attr_accessor :pluralize_table_names
     end
```

Comment:
[here](https://github.com/rails/rails/blob/158c7eb1d61a28452e0aafd1e05314352eea2749/activerecord/lib/active_record/model_schema.rb#L50). Maybe we can just remove the attr_accessor in the test?